### PR TITLE
Add recovery regex config to docs

### PR DESCRIPTION
**Pull request by ShanChathusanda93**

### CODE REVIEW

File: en/docs/learn/password-recovery.md [hunk 1]
```diff
@@ -104,8 +104,10 @@ with email notification.
         You can edit and customize the email template. For more information
         on how to do this, see [Customizing Automated
         Emails](../../learn/customizing-automated-emails)
-.
-    
+
+6. To allow the user's to redirect to a certain location after the password recovery request is completed, change the
+default configuration in **Recovery callback URL regex** into a specific regular expression which will match with the 
+   redirect URL that the user needs to be redirected.
```
Comment (ChamathNS) [inline en/docs/learn/password-recovery.md:110]: Is this formatted correctly?

**Review by ChamathNS — APPROVED** [review]

File: en/docs/learn/password-recovery.md (hunk 1 above)
Comment (ShanChathusanda93) [reply en/docs/learn/password-recovery.md]: Fixed

File: en/docs/learn/password-recovery.md [hunk 2]
```diff
@@ -104,8 +104,10 @@ with email notification.
         You can edit and customize the email template. For more information
         on how to do this, see [Customizing Automated
         Emails](../../learn/customizing-automated-emails)
-.
-    
+
+6. To allow the user's to redirect to a certain location after the password recovery request is completed, change the
```
Comment (ChamathNS) [inline en/docs/learn/password-recovery.md:108]:
```suggestion
6. To allow the users to be redirected to a certain location after the password recovery request is completed, change the
```

File: en/docs/learn/password-recovery.md [hunk 3]
```diff
@@ -104,8 +104,10 @@ with email notification.
...
+default configuration in **Recovery callback URL regex** into a specific regular expression which will match with the
```
Comment (ChamathNS) [inline en/docs/learn/password-recovery.md:109]:
```suggestion
default configuration in **Recovery callback URL regex** to a specific regular expression that will match the 
```

**Review by gomathyk — APPROVED** [review]

File: en/docs/learn/password-recovery.md [hunk 4]
```diff
@@ -104,8 +104,10 @@ with email notification.
         You can edit and customize the email template. For more information
         on how to do this, see [Customizing Automated
         Emails](../../learn/customizing-automated-emails)
-.
-    
+
+6. To allow the users to be redirected to a certain location after the password recovery request is completed, change the
+default configuration in **Recovery callback URL regex** to a specific regular expression that will match the 
+redirect URL that the user needs to be redirected.
```
Comment (gomathyk) [inline en/docs/learn/password-recovery.md:110]:
```suggestion
redirect URL that the user needs to be redirected to.
```

**Review by gomathyk — APPROVED** [review]

File: en/docs/administer/product-level-security-guidelines.md [hunk 5]
```diff
@@ -192,6 +192,15 @@ To enable hostname verification:
 
 By default, XSS attacks are prevented in the latest WSO2 Identity Server versions. This is due to output encoding of the displaying values. 
 
+## Callback URL Regular Expressions
+
+In WSO2 Identity Server, you can specify regular expression patterns for callback URLs. In the recovery scenarios you have the capability to define
```
Comment (gomathyk) [inline en/docs/administer/product-level-security-guidelines.md:197]: Shall we rephrase this as follows? 

For password recovery, you can define a regular expression to validate the callback URL. The default configuration allows any callback URL. Note that if you are using the recovery option, it is highly recommended to define the regular expression that validates and only allows access to specific callback URLs.

File: en/docs/administer/product-level-security-guidelines.md [hunk 6]
```diff
@@ -192,6 +192,15 @@ To enable hostname verification:
...
+a regular expression for validating the callback URL. The default configuration will allow any callback URL. Please note that if you are using 
+recovery scenarios it is highly recommended to define the regular expression which will allow the specific callback URL for the users. 
+
+!!! info
+
+    For more informatoin, see <a href="../../learn/password-recovery">Password Recovery</a>.
```
Comment (gomathyk) [inline en/docs/administer/product-level-security-guidelines.md:203]:
```suggestion
    For more information, see [Password Recovery](../../../learn/password-recovery).
```

File: en/docs/learn/password-recovery.md [hunk 7]
```diff
@@ -104,8 +104,10 @@ with email notification.
         You can edit and customize the email template. For more information
         on how to do this, see [Customizing Automated
         Emails](../../learn/customizing-automated-emails)
-.
-    
+
+6. To allow the users to be redirect to a certain location after the password recovery request is completed, change the
```
Comment (gomathyk) [inline en/docs/learn/password-recovery.md:108]:
```suggestion
6. To allow the users to be redirected to a certain location after the password recovery request is complete, change the
```

**Review by gomathyk — CHANGES_REQUESTED** [review]

**Review by gomathyk — APPROVED** [review]
